posts/how-to-install-sourcegraph-on-mac-with-docker.md

```diff
@@ -0,0 +1,109 @@
+---
+title: How To Install Sourcegraph on macOS with Docker
+author: lisa-tagliaferri
+tags: [tutorial, Docker, installation, Sourcegraph]
+description: Learn how to install Sourcegraph on your Apple Mac with Docker
+image: /headers/sourcegraph-learn-header-6.svg
+imageAlt: Sourcegraph Learn
+socialImage: /headers/sourcegraph-learn-header-6.png
+type: posts
+---
+
+Sourcegraph is a tool that can allow you to search all the code that is important to you, from your own local repositories, to the software you build together with a team, to all the code that is available through open source projects. You can get started using Sourcegraph by using [Sourcegraph Cloud](https://sourcegraph.com/search) with your preferred web browser, or you may want to install Sourcegraph locally or on a cloud server or virtual machine.
```

Comment:
Not sure if we want to go with "Sourcegraph Cloud" or "Sourcegraph cloud" (i.e. not sure if we want to capitalize Cloud or not). Another file uses the lowercase version.

As long as we're consistent I assume it's not highly consequential.

Reply:
Good eye! I missed the changes, the lower-case is the version that the rest of the team has been discussing. Updated the other uses in the commit below.